### ISSUE TITLE
[ Sormas2Sormas ] Sharing and rejecting multiple times results in server error

### ISSUE THREAD

**Issue by AndyBakcsy-she** [hzi-braunschweig/SORMAS-Project#6067]
### Steps to Reproduce
1. Share a newly created case from test-de2 with ownership towards test-de1
2. On Test-de1 reject the case
3. Refresh the page and share one more time
4. Reject again on test-de1 environment
5. Share again from test-de2 environment with ownership
6. On test-de1 accept the case

### Expected Behavior
The case should be accepted and available under Cases Directory

### Actual Behavior
Error messages are displayed after step 3.  On step 6 a server error message was displayed.

### Screenshots
![image](https://user-images.githubusercontent.com/70562142/124474860-b9b83200-dda9-11eb-8d7e-097b19f01f16.png)
![image](https://user-images.githubusercontent.com/70562142/124474951-d18fb600-dda9-11eb-86c8-50a4f5fd9866.png)

### System Details

* Device: windows 10 
* SORMAS version: 1.62.0-snapshot
* Android version/Browser:
* Server URL: test-de1 and test-de2.sormas.netzlink.com
* User Role: NatUser

### Additional Information
this might be related to #4423

**Comment by vidi42** [comment]
Could not reproduce the defect described in this ticket anymore.

Opened a PR for another issue identified during the investigation of this (https://github.com/hzi-braunschweig/SORMAS-Project/pull/6110)

**Comment by AndyBakcsy-she** [comment]
Moving ticket to the Done lane as unreproducible.
